### PR TITLE
[WIP] Allow pagination entities nullable in spec

### PR DESCRIPTION
**Pull request by digitronik**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

I was trying to generate a client with new OpenAPI 5. facing errors like

```
ApiTypeError: Invalid type for variable 'previous'. Required value type is str and passed type was NoneType at ['received_data']['links']['previous']
```

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: docs/openapi.json [hunk 1]
```diff
@@ -1742,16 +1742,20 @@
                 "type": "object",
                 "properties": {
                     "first": {
-                        "type": "string"
+                        "type": "string",
```
Comment (Josca) [inline docs/openapi.json:1745]: This file is autogenerated from gocode annotations so we can not update it manually. But we can do it other way.

Comment (digitronik) [reply docs/openapi.json:1745]: yup. I'm trying to learn it :)

**Comment by Josca** [comment]
@digitronik is it still actual or can we close it?

**Comment by digitronik** [comment]
@Josca my bad... I want to learn it but time :)
closing this as @MichaelMraka  works on openapi5 client. I think it will cover this.

**Pull request closed without merging**